prime-router/metadata/hl7_mapping/ORU_R01/base/note.yml

```diff
@@ -0,0 +1,37 @@
+name: ORU-R01-note
+elements:
+  - name: note-id
+    value: [ '%noteIndex + 1' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-1' ]
+
+  - name: note-source
+    value: [ '''L''' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-2' ]
+
+  - name: note-comment
+    value: [ '%resource' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-3' ]
+
+  - name: note-comment-type
+    value: [ '''RE''' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-4-1' ]
+
+  - name: note-comment-type
+    value: [ '''Remark''' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-4-2' ]
+
+  - name: note-comment-type
+    value: [ '''HL70364''' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-4-3' ]
+
+  - name: note-comment-type
+    value: [ '''C''' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-4-4' ]
+
+  - name: note-comment-type
+    value: [ '''Comment''' ]
+    hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-4-5' ]
+
+  - name: note-comment-type
```

Comment:
can you change the names in this part to be different for each? makes it easier to search the trace

Reply:
Updated